src/components/RadioGroup/Radio.tsx

```diff
@@ -51,13 +51,17 @@ const StyledInput = styled.input`
   }
 `
 
-export type RadioProps = InputHTMLAttributes<HTMLInputElement>
+export interface RadioProps extends InputHTMLAttributes<HTMLInputElement> {
+  $loading?: boolean
```

Comment:
Why the dollar sign here? 🤔 

Ahh I think I know why 😄 

You should destruct the props like this
```tsx
({ loading, ...props }: RadioProps, ref: Ref<HTMLInputElement>)
```
so the input just gets the attributes it needs and the render can be clean as well =) (without `props.loading`)

Reply:
it was not to include it in the radio html input, fixed 🙏